lib/workload/stateless/sequence_run_manager/Makefile

```diff
@@ -11,19 +11,14 @@ lint:
 lint-fix:
 	@black -t py312 ./src --exclude skel --exclude .venv
 
-# full mock suite test pipeline - install deps, bring up compose stack, wait a bit, run suite, bring down compose stack
-test: install up wait suite down
-
-# heuristic wait 8s to make sure the database is fully up
-# todo we could improve this
-wait:
-	@sleep 8
+# full mock suite test pipeline - install deps, bring up compose stack, run suite, bring down compose stack
+test: install up suite down
 
 suite:
 	@python manage.py test
 
 up:
-	@docker compose up -d
+	@docker compose up --wait -d
```

Comment:
@victorskl This `--wait` flag should do the wait for the database up. Added a startup period on the docker-compose for the healthcheck